### DISCUSSION TITLE
Making multiplayer chat box toggle-able

### DISCUSSION THREAD

**Discussion by FoomZz** [ppy/osu#15154] [Ideas]
As it is right now, or at least as far as I'm aware, when playing, the chat box is on the screen throughout the entire map.  The problem I have with this is that it's 1. distracting, and 2. whilst minor, it interferes/covers gameplay. My proposal is to have a feature to hide chat during gameplay, making it toggle-able via either key toggle, or an through an option in settings. Or if this isn't viable, maybe resizing the chat box so that it doesn't overlap gameplay.

**Comment by bdach** [comment]
It already can be toggled by hiding the HUD (Shift+Tab). Unsure if that is satisfactory.

**Reply by FoomZz** [reply]
IMO it isn't satisfactory, as i'd still like to see the hiterrormeter, accuracy and etc.

**Reply by cj4m** [reply]
I'm on the same page as FoomZz, I really enjoy the new full size leaderboard and would like to keep it up _(or maybe move it in the future with the skin editor?)_ but the chatbox is pretty bothersome. I'd love to see the chatbox perform similar to how the leaderboard does in stable, where you can toggle it with tab to only appear during breaks! During actual gameplay it doesn't really make sense to have the chatbox always visible as reading it and playing at the same time is not exactly easy and only causes distractions if spectators are talking in the chat or if the chatbox is on top of a circle.
That being said, I do like the minimal leaderboard when you hide the HUD, but I feel like it's valid to have options - hidden HUD is the minimal scoreboard, and hitting tab (or whatever keybind) would just hide the chatbox on the fullsize leaderboard so elements like the hit error meter is always visible for those that enjoy it. 

I don't believe this would take somebody out of the multiplayer experience, it just felt a little unnecessary as sometimes people don't talk at all.

**Reply by peppy** [reply]
We can probably start by collapsing it better in cases there is no chat.

**Reply by localcake** [reply]
It would be preferable if there was an option to hide the chat and leaderboard completely as I play multiplayer and there's always a few players that complain about it.
